commune.lic

```diff
@@ -0,0 +1,367 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#commune
+=end
+
+custom_require.call(%w(common common-items common-arcana drinfomon))
+
+class Commune
+  include DRC
+  include DRCI
+  include DRCA
+
+  ######################## ITEM ACTIONS ########################
+
+  def get_cleric_item(item)
+    case bput("get #{item} from my #{@theurgy_supply_container}", 'You get', 'You are already holding', 'I could not find', 'What were you referring to')
+	when 'You get' || 'You are already holding'
+	  return true
+	else
+	  return false
+	end
+  end
+
+  def stow_cleric_item(item)
+    case bput("put my #{item} in my #{@theurgy_supply_container}", 'You put', 'What were you referring to')
+	when 'You put'
+	  return true
+	else
+	  return false
+	end
+  end
+
+  def empty_cleric_hands(hand='both')
```

Comment:
Why does `empty_cleric_hands` take a parameter if it is never called with one?

Reply:
Good question. I eliminated the parameter and made empty_cleric_hands simply empty right and empty left.